roles/activation_keys/tasks/main.yml: yaml[line-length]: Line too long

##### SUMMARY
ansible-lint warns about too long lines:
```
roles/activation_keys/tasks/main.yml:11: yaml[line-length]: Line too long (193 > 160 characters) 
roles/activation_keys/tasks/main.yml:12: yaml[line-length]: Line too long (184 > 160 characters) 
```


##### ISSUE TYPE
 - Bug Report

##### ANSIBLE VERSION
<!-- Paste, BELOW THIS COMMENT, verbatim output from "ansible --version"-->
```

```

##### COLLECTION VERSION
<!-- Paste, BELOW THIS COMMENT, verbatim output from "ansible-galaxy collection list"-->
```

```

##### KATELLO/FOREMAN VERSION
<!-- Paste, BELOW THIS COMMENT, verbatim output from "rpm -q rubygem-katello foreman"-->
```

```

##### STEPS TO REPRODUCE
<!-- For bugs, show exactly how to reproduce the problem, using a minimal test-case. -->

<!-- Paste example playbook below -->
```yaml

```

##### EXPECTED RESULTS
<!-- What did you expect to happen when running the steps above? -->

##### ACTUAL RESULTS
<!-- What actually happened? If possible run with extra verbosity (-vvvv) and diff (--diff) -->
<!-- Please also include check mode (--check --diff) output if the API returns an error -->
<!-- Be sure to mask any sensitive information -->

<!--- Paste verbatim command output between quotes below -->
```

```

Hmm... removing those two lines does not change the result of the linting. For me, yamllint still says: everything ok!

```
$ cat .yamllint                                                          
---
ignore: |
  tests/test_playbooks/fixtures/
$ yamllint -c .yamllint roles/activation_keys/tasks/main.yml && echo "ok"
ok
```

I have to explicitly set `max` to `160` to get errors:

```
$ cat .yamllint                                                                           
---
rules:
  line-length:
    max: 160
ignore: |
  tests/test_playbooks/fixtures/
$ yamllint -c .yamllint roles/activation_keys/tasks/main.yml                              
roles/activation_keys/tasks/main.yml
  11:161    error    line too long (193 > 160 characters)  (line-length)
  12:161    error    line too long (184 > 160 characters)  (line-length)
```

The messages you see above or in #1948 seem to come from ansible-lint.

--- a/roles/activation_keys/tasks/main.yml
+++ b/roles/activation_keys/tasks/main.yml
@@ -8,8 +8,18 @@
     organization: "{{ item.organization | default(foreman_organization) }}"
     name: "{{ item.name }}"
     description: "{{ item.description | default(omit) }}"
-    lifecycle_environment: "{{ item.lifecycle_environment | default(omit) if (item.content_view_environments | default([]) | length > 0) else item.lifecycle_environment | default('Library') }}"
-    content_view: "{{ item.content_view | default(omit) if (item.content_view_environments | default([]) | length > 0) else item.content_view | default('Default Organization View') }}"
+    lifecycle_environment: >-
+      {{
+        item.lifecycle_environment | default(omit)
+        if (item.content_view_environments | default([]) | length > 0)
+        else item.lifecycle_environment | default('Library')
+      }}
+    content_view: >-
+      {{
+        item.content_view | default(omit)
+        if (item.content_view_environments | default([]) | length > 0)
+        else item.content_view | default('Default Organization View')
+      }}
     content_view_environments: "{{ item.content_view_environments | default(omit) }}"
     host_collections: "{{ item.host_collections | default(omit) }}"
     subscriptions: "{{ item.subscriptions | default(omit) }}"

